src/um/winbase.rs

```diff
@@ -2155,10 +2155,46 @@ extern "system" {
         lpBuffer: LPWSTR,
         pcbBuffer: LPDWORD
     ) -> BOOL;
```

Comment:
Right here is where that block of constants from below should go.

Reply:
But the constant will be included in the extern "system" block, is it what we want ? Where constant should be declared ? Why should I move them there ? Thx

Reply:
You would separate the `extern "system"` block into two chunks.

Reply:
The constants should be there, because that is where they are in the Windows SDK headers, and the order in the source files should match the order in the header files.

Reply:
Done. Same order, same position as Windows SDK header files. Thx